src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -3,13 +3,13 @@
 import mate.academy.model.User;
 
 public class AuthenticationService {
+    private User user;
 
     public boolean login(String email, String password) {
         UserService userService = new UserService();
-        // Find the user by email
-        User user = userService.findByEmail(email);
 
-        // Check if the user exists and the provided password matches the user's password
-        return user != null && user.getPassword().equals(password);
+        this.user = userService.findByEmail(email);
+
+        return this.user != null && this.user.getPassword().equals(password);
```

Comment:
check this out to know where we need `this`
https://stackoverflow.com/a/2411283
when we don't, no need to use it